app/validators/claim/base_claim_validator.rb

```diff
@@ -91,7 +92,8 @@ def validate_transfer_court(force: false)
 
   def validate_transfer_case_number
     return if @record.errors[:transfer_case_number].present?
-    validate_pattern(:transfer_case_number, CASE_NUMBER_PATTERN, 'invalid')
+    validate_pattern(:transfer_case_number, URN_PATTERN, 'invalid')
+    validate_pattern(:transfer_case_number, CN_PATTERN, 'format') if looks_like_a_case_number?(:transfer_case_number)
```

Comment:
use the same message `invalid` and change that message as [i mention below](https://github.com/ministryofjustice/Claim-for-Crown-Court-Defence/pull/3231/files#r397717110)

Reply:
👍